Cargo.toml

```diff
@@ -73,6 +73,7 @@ test-log = { version = "0.2.16", features = [
 ], default-features = false }
 tokio-stream = "0.1.16"
 tempfile = "3.13.0"
+testcontainers-modules = { version = "0.11.2", features = ["postgres"] }
```

Comment:
this dep is already in dependencies line 53, why having it also in dev-dependencies ?

Reply:
parce que j'ai pas rebase correctement.

Mais c'est pas plutôt à garder dans dev-dependencies?

Reply:
oui en effet, t'as raison, uniquement dev-dependencies :+1: